packages/node/src/methods/method-handler.ts

```diff
@@ -0,0 +1,81 @@
+import { Node } from "@counterfactual/common-types";
+import EventEmitter from "eventemitter3";
+
+import { Channels } from "../channels";
+import { NodeMessage } from "../node";
+import { IMessagingService } from "../service-interfaces";
+
+import { addAppInstance, install, proposeInstall } from "./installations";
+import { addMultisig, createMultisig } from "./multisig-creation";
+
+export class MethodHandler {
+  private METHODS = new Map();
```

Comment:
@cf19drofxots Any particular reason for using upper-case here?

Reply:
Wasn't a dynamic map at first. Casing is stale, will update.